libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -431,3 +431,9 @@ MQTTStatus_t MQTT_DeserializeAck( const MQTTPacketInfo_t * const pIncomingPacket
 
     return MQTTSuccess;
 }
+
+MQTTStatus_t MQTT_GetIncomingPacketTypeAndLength( MQTTTransportRecvFunc_t readFunc,
```

Comment:
This needs to implemented. The length computation logic can be directly borrowed for V4. Only change needed is instead of getNextByte(), use transport_read with one byte.

Reply:
Implemented as part of #911